src/test/java/redis/clients/jedis/JedisClientSideCacheTest.java

```diff
@@ -26,45 +28,83 @@ public void tearDown() throws Exception {
     jedis.close();
   }
 
-  private static final JedisClientConfig configForCache = DefaultJedisClientConfig.builder()
-      .resp3().socketTimeoutMillis(20).password("foobared").build();
+  private static final JedisClientConfig clientConfig = DefaultJedisClientConfig.builder().resp3().password("foobared").build();
 
   @Test
   public void simple() {
-    try (JedisClientSideCache jCache = new JedisClientSideCache(hnp, configForCache)) {
+    try (JedisClientSideCache jCache = new JedisClientSideCache(hnp, clientConfig)) {
       jedis.set("foo", "bar");
       assertEquals("bar", jCache.get("foo"));
       jedis.del("foo");
-      assertNull(jCache.get("foo"));
+      assertThat(jCache.get("foo"), Matchers.oneOf("bar", null)); // ?
```

Comment:
Can this be achieved without adding this import?

Reply:
@chayim We'd have to write a "utility" method.
But why bother? We already have these as our test dependency and being used in some other tests.

Reply:
General desire to have fewer dependencies.. even if it's just for test. Not a hard requirement.